routes/middlewares/user/jwt/getUserFromToken.ts

```diff
@@ -9,14 +9,18 @@ import Teacher from '@Model/teacher.model';
 import User from '@Model/user.model';
 
 const getUserFromToken = async (req: Request, res: Response, next: NextFunction) => {
-  const pk = res.locals.user.pk;
+  const verifiedToken = res.locals.user;
```

Comment:
```suggestion
  const tokenDecoded = res.locals.user;
```
verifyToken 이후 res.locals.user는 토큰이 아니라 디코딩 된 값이라 추후에 알아보기 어려울 듯